docs/sources/query/metric_queries.md

```diff
@@ -153,3 +153,14 @@ Examples:
       or
     vector(0) # will return 0
     ```
+
+## Probabilistic aggregation
+
+LogQL also supports a probabilistic `topk` approximation that is a drop-in replacement when `topk` hits the maximum series limit.
```

Comment:
The current text assumes that users are already familiar with what topk is, but we don't really do a good job of defining it in the documentation.

```suggestion
The `topk` keyword lets you find the largest 1,000 elements in a data stream by sample size. When  `topk` hits the maximum series limit, LogQL also supports using a probable approximation; `approx_topk`  is a drop-in replacement when `topk` hits the maximum series limit.
```